practice-two/src/services/customerService.js

```diff
@@ -1,12 +1,58 @@
-// import { BASE_URL, PATH } from "@constants";
+import { BASE_URL, PATH } from '@constants';
+import { mutate } from 'swr';
 
+// Get customer list
 const getAllCustomerService = async (url) => {
   const response = await fetch(url);
   if (!response.ok) {
-    throw new Error("An error occurred while fetching the data.");
+    throw new Error('An error occurred while fetching the data.');
   }
   const data = await response.json();
```

Comment:
return both error and data for api services

Reply:
Because SWR supports how to check errors, so I change the way return both data and error to handle create, update and delete
![image](https://github.com/datdragon777/React-Training/assets/74806826/5e6837c8-0577-4927-921a-63fc4b7018e6)